### PR TITLE
Transfer playback when changing device

### LINKED ISSUE #366: Cannot change device.
After running spotifyd and then spt, I am unable to see my computer as an option for devices. This seems to be a spotifyd issue, but I figured I would check here first. I can see my tv, and phone as options (but I cannot switch the playback device as Enter will not change the device for playback) but  my computer will not show up. If I launch the spotify client from the AUR then my computer shows as an option but I again I am unable to switch the playback device. If I change the playback device via my phone though, spotify-tui correctly recognizes a new playback device has been chosen.

### PR DESCRIPTION
**Pull request by TimotheeGerber**
> Warning: This PR will not work before [PR #90 in `rspotify`](https://github.com/ramsayleung/rspotify/pull/90) has been merged and a new version has been released. If you want to test it, you can clone my patched version of `rspotify` and ask `cargo` to use it instead of the one in `crates.io` by adding the following in your `Cargo.toml`:
>
> ```toml
> [patch.crates-io]
> rspotify = { path = "../relative/path/to/patched_rspotify" }
> ```

Currently, selecting another device changes the device in the internal representation but does not transfer playback to the selected device. This is counter intuitive and lead to strange situations.

Imagine I have 2 devices, `receiver_1` and `receiver_2`, and I am currently playing a song on `receiver_1`. If I select `receiver_2`, `receiver_1` continues to play the song. Nevertheless, commands (pause, next, previous, etc) are sent to `receiver_2`. So, a pause command will not pause the song because `receiver_1` does not get the command. `receiver_2` gets the command and cannot do anything with it.

This PR changes that behavior and transfer playback when selecting a device.

I hope that the modifications I have made are OK. Don't hesitate to comment them if needed!

Closes #366

### CODE REVIEW

File: src/network.rs [hunk 1]
```diff
@@ -1144,7 +1144,17 @@ impl<'a> Network<'a> {
     }
   }
 
-  async fn set_device_id_in_config(&mut self, device_id: String) {
+  async fn transfert_playback_to_device(&mut self, device_id: String) {
```
Comment (Rigellute) [inline src/network.rs:1147]: Nice 👍

File: README.md [hunk 2]
```diff
@@ -421,7 +421,7 @@ This table shows all that is possible with the Spotify API, what is implemented
...
 | current_playback                                  | Yes              | Get Information About The User’s Current Playback                                                                                                            | Yes        |
 | current_playing                                   | No               | Get the User’s Currently Playing Track                                                                                                                       | No         |
-| transfer_playback                                 | No               | Transfer a User’s Playback                                                                                                                                   | No         |
+| transfer_playback                                 | Yes              | Transfer a User’s Playback                                                                                                                                   | Yes        |
```
Comment (Rigellute) [inline README.md:424]: Thanks for updating this

**Comment by TimotheeGerber** [comment]
A new version of `rspotify` has been released this summer and `spotify-tui` already use it. This PR can now be integrated. It still works and will improve playback transfer for people using multiple receivers.

**Comment by Rigellute** [comment]
@TimotheeGerber awesome, thanks

**Pull request merged**